api/src/services/files.ts

```diff
@@ -71,19 +72,62 @@ export class FilesService extends ItemsService {
 			payload.type = 'application/octet-stream';
 		}
 
+		const tempFilenameDisk = 'temp_' + payload.filename_disk;
+
 		try {
-			await storage.location(data.storage).write(payload.filename_disk, stream, payload.type);
+			await disk.write(tempFilenameDisk, stream, payload.type);
+
+			// Check if the file was truncated (if the stream ended early)
+			if (stream.truncated === true) {
```

Comment:
BusBoy just adds the property `truncated` to the readable string, how can I fix type issues with this?